        <html><head></head><body>
        <h1>Report for task on toolchain
            <a href="index.html">android</a>
        </h1>
        <p>Return code: 1</p>
        <p>Time to run: 00:00:43</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_android -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data task
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 1751 packages in cache
===> sudo -u tuscan makepkg --nobuild --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: task 2.4.4-1 (Fri Nov 13 23:46:20 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Packages (4) jsoncpp-1.6.5-1  libxml2-2.9.2-2  shared-mime-info-1.5-1  cmake-3.3.2-3

Total Installed Size:  39.83 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
checking available disk space...
installing libxml2...
Optional dependencies for libxml2
    python2: python bindings to libxml
installing shared-mime-info...
installing jsoncpp...
installing cmake...
Optional dependencies for cmake
    qt5-base: cmake-gui
    libxkbcommon-x11: cmake-gui
==> Retrieving sources...
  -> Downloading task-2.4.4.tar.gz...
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
  -> Extracting task-2.4.4.tar.gz with bsdtar
==> Sources are ready.
Copied source directory to /sources/task
Traceback (most recent call last):
  File "/build/make_package.py", line 215, in <module>
    main()
  File "/build/make_package.py", line 212, in main
    result = copy_and_build(args)
  File "/build/make_package.py", line 118, in copy_and_build
    copytree(permanent_source_dir, build_dir)
  File "/usr/lib/python3.5/shutil.py", line 353, in copytree
    raise Error(errors)
shutil.Error: [('/sources/task/src/task-2.5.0/src/cal', '/tmp/task/src/task-2.5.0/src/cal', "[Errno 2] No such file or directory: '/sources/task/src/task-2.5.0/src/cal'"), ('/sources/task/src/task-2.5.0/src/tw', '/tmp/task/src/task-2.5.0/src/tw', "[Errno 2] No such file or directory: '/sources/task/src/task-2.5.0/src/tw'"), ('/sources/task/src/task-2.5.0/src/calendar', '/tmp/task/src/task-2.5.0/src/calendar', "[Errno 2] No such file or directory: '/sources/task/src/task-2.5.0/src/calendar'")]></pre></code>
        </body></html>
